src/Components/RoomForm/RoomForm.js

```diff
@@ -7,33 +7,40 @@ function RoomForm() {
   const [ roomname, setRoomname ] = useState('')
   const [ rules, handleRules ] = useState([])
   const [ potentialRule, setRule ] = useState('')
-  const roomID = uniqid()
-  // useEffect()
+  const [ roomID, setroomID ] = useState(uniqid())
+
+  const messagesEndRef = React.createRef()
+
+  const scrollToBottom = () => {
+    messagesEndRef.current.scrollIntoView({ behavior: 'smooth' })
+  }
```

Comment:
what does createRef do?

Reply:
https://reactjs.org/docs/refs-and-the-dom.html this is a great article, but basically it creates a reference point on the page for use with:
* Managing focus, text selection, or media playback.
* Triggering imperative animations.
* Integrating with third-party DOM libraries.

Reply:
WOW I'll take a look at this